internal/operators/ocs/ocs_operator_test.go

```diff
@@ -32,20 +32,6 @@ var _ = Describe("Ocs Operator", func() {
 			Inventory: Inventory(&InventoryResources{Cpus: 12, Ram: 32 * conversions.GiB,
 				Disks: []*models.Disk{
 					{SizeBytes: 20 * conversions.GB, DriveType: "HDD", ID: diskID1}}})}
-
-		masterWithLessCPU = &models.Host{Role: models.HostRoleMaster, InstallationDiskID: diskID1,
```

Comment:
Won't this be caught by GetHostRequirements

Reply:
GetHostRequirements returns the operator's requirement for the operator, which is checked on a higher level, including all other operator requirements

Reply:
Can you clarify why this test is no longer required?

Reply:
The test that used this host was ValidateHost, as we are removing the check of resources(CPU/RAM) from ValidateHost so this variable would become unused.
We can move this test into GetHostRequirements which return the minimum CPU/RAM required by this host depending on the number of disks it has.

Reply:
Yes. Please move these tests to GetHostRequirements.